docs/man/man1/kubectl-drain.1

```diff
@@ -33,6 +33,10 @@ will make the node schedulable again.
 
 .SH OPTIONS
 .PP
+\fB\-\-delete\-local\-data\fP=false
+    Continue even if there are pods with local data (EmptyDir).
```

Comment:
I don't think this should just say "local data" since HostDir is also local data. Maybe say "Continue even if there are pods using emptyDir (local data that will be deleted when the node is drained)."